src/routes.js

```diff
@@ -305,12 +305,11 @@ class Routes extends React.Component<Props> {
                 />
                 <Route
                   // NOTE(@mxstbr): This custom path regexp matches threadId correctly in all cases, no matter if we prepend it with a custom slug or not.
-                  // Imagine our threadId is "id-123-id" (similar in shape to an actual UUID)
-                  // - /id-123-id => id-123-id, easy start that works
-                  // - /some-custom-slug~id-123-id => id-123-id, custom slug also works
-                  // - /~id-123-id => id-123-id => id-123-id, empty custom slug also works
-                  // - /some~custom~slug~id-123-id => id-123-id, custom slug with delimiter char in it (~) also works! :tada:
-                  path="/:communitySlug/:channelSlug/(.*~)?:threadId"
+                  // Imagine our threadId is "id-123-123-123-id" (similar in shape to an actual UUID)
+                  // - /id-123-123-123-id => id-123-123-123-id, easy start that works
+                  // - /some-custom-slug-id-123-123-123-id => id-123-123-123-id, custom slug also works
+                  // - /-id-123-123-123-id => id-123-123-123-id, empty custom slug also works
+                  path="/:communitySlug/:channelSlug/(.*?)(\w*-\w*-\w*-\w*-\w*)"
```

Comment:
Fairly sure this will break for all firebase-imported threads from way back in the day.